pymagicc/api.py

```diff
@@ -147,27 +152,65 @@ def out_dir(self):
             return None
         return join(self.root_dir, "out")
 
-    def run(self, only=None):
+    def run(self, scenario=None, only=None, include_parameters=None, **kwargs):
         """
         Run MAGICC and parse the output.
 
         Parameters
         ----------
+        scenario : :obj:`pymagicc.io.MAGICCData`
+            Scenario to run. If None MAGICC will simply run with whatever config has
+            already been set.
+
         only
             If not None, only extract variables in this list
 
+        include_parameters
```

Comment:
Whatever "none" means what is set in the config file, right? Maybe this could be spelled out, if i understood it correctly?

Reply:
Not quite. Basically you need a default which says, don't change the existing config on my `MAGICC` instance. i.e. if you already said somewhere else:

```python
magicc = MAGICC6()
magicc.set_config(out_parameters=0)
```

then you don't want `magicc.run()` to silently set `out_parameters`, you want it to just not change any of the existing config. Hence you need the default to be something which is neither `True` nor `False` hence I went for `None`. Line 208 is where it's used (I'm not sure how better to explain this in the docs...):

```python
if include_parameters is not None:
    kwargs["out_parameters"] = 1 if include_parameters else 0
# else don't alter whatever the current config value of out_parameters is
```

Reply:
suggestions welcome

Reply:
Hmm, that's still confusing to me ... 

Basically it's just syntactic sugar for haveing "out_parameters` = 1 as a keyword argument in the run function?

Then we could also drop it, and put a reminder in the doc string that, for example, out_parameters=0 will prevent reading out parameters?

Reply:
> Basically it's just syntactic sugar for haveing "out_parameters` = 1 as a keyword argument in the run function?

Yep